runtime/doc/treesitter.txt

```diff
@@ -57,151 +57,6 @@ loaded: >lua
     vim.treesitter.language.add('python', { path = "/path/to/python.wasm" })
 <
 
-==============================================================================
-TREESITTER TREES                                             *treesitter-tree*
```

Comment:
Note: The ordering of these sections was changed slightly (moved below) because otherwise all sections after them (until the next generated section) were deleted. Is there an easy way to fix this? If not hopefully the reordering is nbd

Reply:
We need to generate more; then this will work itself out. I am fine with the new order. (Generally, high-level/Neovim-wide sections should come first; low-level API later.)